### ISSUE TITLE
Extension 2.25.3 terraform-ls not a static binary

### ISSUE THREAD

**Issue by stephanwehr** [hashicorp/vscode-terraform#1346]
## Versions
2.25.3

### Extension
hashicorp.terraform

### VS Code
1.75.1

### Operating System
Windows 10

### WSL
```
wsl --status
Standard Distribution: Ubuntu
Standardversion: 2
Kernel-Version: 5.10.102.1

lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.5 LTS
Release:        20.04
Codename:       focal
```

### Expected Behavior
The binary terraform-ls should be static
```
ldd .vscode-server/extensions/hashicorp.terraform-2.25.2-linux-x64/bin/terraform-ls
        not a dynamic executable
```

### Actual Behavior
The binary terraform-ls is not a static binary
```
ldd .vscode-server/extensions/hashicorp.terraform-2.25.3-linux-x64/bin/terraform-ls
.vscode-server/extensions/hashicorp.terraform-2.25.3-linux-x64/bin/terraform-ls: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by .vscode-server/extensions/hashicorp.terraform-2.25.3-linux-x64/bin/terraform-ls)
.vscode-server/extensions/hashicorp.terraform-2.25.3-linux-x64/bin/terraform-ls: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by .vscode-server/extensions/hashicorp.terraform-2.25.3-linux-x64/bin/terraform-ls)
        linux-vdso.so.1 (0x00007ffe311ec000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f9e24de5000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f9e24fe2000)
```

**Comment by dbanck** [comment]
Thank you for reporting the issue. I can reproduce the behavior.

We'll cut a patch release with static binaries today to fix this.

**Comment by dbanck** [comment]
We just released the bug fix for this as part of the `v2.25.4` extension release.
